java/code/src/com/redhat/rhn/frontend/xmlrpc/sync/content/RMTContentSyncSource.java

```diff
@@ -70,6 +70,8 @@ public SCCClient getClient(String uuid, Path loggingDir) throws ContentSyncSourc
 
             SCCConfig config = new SCCConfigBuilder()
                     .setUrl(url)
+                    .setUsername(credentials.getUsername())
+                    .setPassword(credentials.getPassword())
```

Comment:
@lucidd Seems that we normally do not need username and password for RMT connection, but now the username is used in writeCache. Not sure if we should also set the password or not?

Reply:
:+1: I will remove the password